### PR TITLE
Prepare 1.6.0

### PR DESCRIPTION
**Pull request by daniel-mohemian**
Includes RSA-OAEP-256 support (#135) by @stulevine.

### CODE REVIEW

File: JOSESwift/Sources/CryptoImplementation/RSA.swift [hunk 1]
```diff
@@ -60,22 +60,20 @@ internal extension AsymmetricKeyAlgorithm {
...
-     - parameters:
-        - publicKey: SecKey the publicKey used to acquire the SecKey block size
-     - returns: Int the maximum length allowed
-     discussion:
-     - RSA1_5: For detailed information about the allowed plain text length for RSAES-PKCS1-v1_5, please refer to the RFC(https://tools.ietf.org/html/rfc3447#section-7.2)
-     - RSAOAEP256: For detailed information about the allowed plain text length for RSA-OAEP, please refer to the RFC(https://tools.ietf.org/html/rfc3447#section-7.1)
-    */
+
+    /// This method returns the maximum message length allowed for an `AsymmetricKeyAlgorithm`.
+    /// - Parameter publicKey: the publicKey used with the algorithm
```
Comment (carol-mohemian) [inline JOSESwift/Sources/CryptoImplementation/RSA.swift]: `the publicKey` -> `The publicKey` and missing period.

File: JOSESwift/Sources/CryptoImplementation/RSA.swift [hunk 2]
```diff
@@ -60,22 +60,20 @@ internal extension AsymmetricKeyAlgorithm {
...
+    /// - Returns: The maximum messeage length allowed for use with the algorithm
```
Comment (carol-mohemian) [inline JOSESwift/Sources/CryptoImplementation/RSA.swift]: `messeage` -> `message` and missing period.

File: JOSESwift/Sources/CryptoImplementation/RSA.swift [hunk 3]
```diff
@@ -60,22 +60,20 @@ internal extension AsymmetricKeyAlgorithm {
...
+    ///
+    /// - RSA1_5: For detailed information about the allowed plain text length for RSAES-PKCS1-v1_5, please refer to [RFC-3447, Section 7.2](https://tools.ietf.org/html/rfc3447#section-7.2).
```
Comment (carol-mohemian) [inline JOSESwift/Sources/CryptoImplementation/RSA.swift]: Could you insert a line break?

File: JOSESwift/Sources/CryptoImplementation/RSA.swift [hunk 4]
```diff
@@ -60,22 +60,20 @@ internal extension AsymmetricKeyAlgorithm {
...
+    /// - RSAOAEP256: For detailed information about the allowed plain text length for RSA-OAEP, please refer to [RFC-3447, Section 7.1](https://tools.ietf.org/html/rfc3447#section-7.1).
```
Comment (carol-mohemian) [inline JOSESwift/Sources/CryptoImplementation/RSA.swift]: Could you insert a line break?

File: JOSESwift/Sources/CryptoImplementation/RSA.swift [hunk 5]
```diff
@@ -60,22 +60,20 @@ internal extension AsymmetricKeyAlgorithm {
...
     func maxMessageLength(for publicKey: SecKey) -> Int {
         let k = SecKeyGetBlockSize(publicKey)
         switch self {
         case .RSA1_5:
             return (k - 11)
         case .RSAOAEP256:
-            /// hash length calculation based on SHA-256 algorithm
+            // maximum plaintext length is based on the hash length of SHA-256
```
Comment (carol-mohemian) [inline JOSESwift/Sources/CryptoImplementation/RSA.swift]: `maximum` -> `Maximum`

**Comment by daniel-mohemian** [comment]
Thx @carol-mohemian, I updated the doc comment. 👍

File: CHANGELOG.md [hunk 6]
```diff
@@ -24,6 +24,10 @@ merge to master and move them to a respective version upon release.
 
 Add your changes here.
 
+## [1.6.0] - 2019-01-24
```
Comment (carol-mohemian) [inline CHANGELOG.md]:
```suggestion
## [1.6.0] - 2019-01-25
```

**Pull request merged**